WcaOnRails/app/controllers/admin_controller.rb

```diff
@@ -98,7 +98,6 @@ def leader_senior_voters
 
   private def voters(users, filename)
     csv = CSV.generate do |line|
-      line << ["id", "email", "name"]
 
```

Comment:
If I get it right, Travis isn't passing because it doesn't like this empty line here.